scripts/generate-google3-docs/resource-reference/templates/compute_computemanagedsslcertificate.tmpl

```diff
@@ -0,0 +1,48 @@
+{{template "headercomment.tmpl" .}}
```

Comment:
Looks like not all the sections in the reference doc template is here. You can probably just copy https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/generate-google3-docs/resource-reference/templates/compute_computesslcertificate.tmpl over.